vendor/github.com/vmware/govmomi/ovf/cim.go

```diff
@@ -16,11 +16,14 @@ limitations under the License.
 
 package ovf
 
+import (
+	"github.com/vmware/govmomi/vim25/types"
```

Comment:
This is the same package we are vendoring here. This folder should be copied here (to the vendored package). I don't see it being added in this PR

Also, what exact types are used here? If you need just a struct or two, just copy it.